mod-orders-storage/schemas/purchase_order.json

```diff
@@ -66,14 +61,6 @@
       "type": "object",
       "$ref": "renewal.json"
     },
-    "totalEstimatedPrice": {
```

Comment:
probably worth mentioning somewhere that these fields must be removed from UI as well.

Reply:
The UI works based on content of `composite_purchase_order.json` schema. Both `totalEstimatedPrice` and `totalItems` are still there and going to be calculated all the time depending on PO Line content by `mod-orders`. Also these values are not supposed to be stored in the database. So to avoid any confusions the best approach is to remove them from the storage schema :)